.github/ISSUE_TEMPLATE/Bug-Report.yml

```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report for MCprep! We just need some info
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+    - type: textarea
+      id: steps-to-reproduce
+      attributes:
+        label: What steps are needed to produce the bug?
+        placeholder: Tell us what you did!
+        value: "Step 1: "
+      validations:
+        required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: MCprep Version
+      description: What version of MCprep are you using?
+      options:
+        - 3.3.1 (Default)
```

Comment:
I'm also thinking maybe we should make this not be an explicit list of options too since people can always look up their version, if it's required they'll enter it.